proto/cosmos/offchain/v1alpha1/offchain.proto

```diff
@@ -0,0 +1,15 @@
+syntax="proto3";
+
+package cosmos.offchain.v1alpha1;
+
+option go_package="github.com/cosmos/cosmos-sdk/x/auth/offchain";
+
+import "gogoproto/gogo.proto";
+
+// MsgSignData defines an arbitrary, general-purpose, off-chain message
+message MsgSignData {
```

Comment:
```suggestion
message SigData {
```

Reply:
1. we shouldn't prefix it with `Msg` because that would confuse with state machine messages  / transaction messages.
2. Sig (signature) is better than Sign (signing)

Reply:
If you change the name as part of this PR, could you also adapt ADR 036 accordingly? We have Amino sirning with type `sign/MsgSignData` there.

Since this sits in the `msg` field of a transaction, the current naming makes perfect sense to me. The SDK should just not process `MsgSignData`s.

Reply:
What's the end decision with this? Should we remove `Msg` or not?